corgidrp/walker.py

```diff
@@ -237,6 +243,8 @@ def guess_template(dataset):
             recipe_filename = "l1_to_boresight.json"
         elif image.pri_hdr['VISTYPE'] == "FFIELD":
             recipe_filename = "l1_flat_and_bp.json"
+        elif image.pri_hdr['VISTYPE'] == "ABSFLXFT" or image.pri_hdr['VISTYPE'] == "ABSFLXBT":
```

Comment:
If this is an L2b recipe, the logic for it should be further down, not in thee L1 section